tests/MongoDB.Driver.Tests/MongoClientTests.cs

```diff
@@ -206,6 +210,33 @@ public void UsesSameClusterWhenReadPreferenceTagsAreTheSame()
             databaseNames.Should().Equal(operationResult["databases"].AsBsonArray.Select(record => record["name"].AsString));
         }
 
+        // will probably need to move this test?
+        [SkippableTheory]
+        [ParameterAttributeData]
+        public void ReadPreference_should_not_be_sent_to_standalone_server(
+            [Values(false, true)] bool async)
+        {
+            RequireServer.Check().ClusterType(ClusterType.Standalone);
+            var eventCapturer = new EventCapturer().Capture<CommandStartedEvent>(e => e.CommandName.Equals("find"));
+            using (var subject = new DisposableMongoClient(CreateDisposableClient(eventCapturer).WithReadPreference(ReadPreference.Secondary)))
+            {
+                var database = subject.GetDatabase(DriverTestConfiguration.DatabaseNamespace.DatabaseName);
+                var collection = database.GetCollection<BsonDocument>(DriverTestConfiguration.CollectionNamespace.CollectionName);
+
+                if (async)
+                {
+                    collection.FindAsync("{x: 2}").GetAwaiter().GetResult();
```

Comment:
Probably it's better to use the following format(with spaces between all parts) for json: `{ x : 2 }`

Reply:
Done.